docs/reference/configuration.md

```diff
@@ -318,9 +318,11 @@ area: {
 
 **Type**: `number` (for `decodeSingle` only)
 
-**Default**: Original image dimensions (when omitted)
+**Default**: `800` (for `decodeSingle`); original image dimensions when using `init()`
 
-**Description**: When using `decodeSingle` with file paths, scales the input image so that the longest side (width or height) equals this value, maintaining aspect ratio. When omitted, the original image dimensions are used without any scaling.
+**Description**: When using `decodeSingle` with file paths, scales the input image so that the longest side (width or height) equals this value, maintaining aspect ratio.
```

Comment:
similar to previous comment, this seems to indicate that this option only works with decodeSingle, however the line directly above this one indicates that it does work with decodeSingle or with init/start.  The only differentiation should be that decodeSIngle has a default of 800, otherwise it is original image dimensions if omitted.